cilium-cli/utils/features/features.go

```diff
@@ -287,7 +303,7 @@ func ExtractTunnelFeatureFromVersionedConfigMap(ciliumVersion semver.Version, cm
 		return Status{
 			Enabled: enabled,
 			Mode:    proto,
-		}
+		}, getTunnelPortFeature(proto)
```

Comment:
That feels like a hack :thinking: I wonder if we shouldn't just update `Status` instead?